src/content/changes/highlights/2021-11-12.md

```diff
@@ -0,0 +1,32 @@
+---
+date: 2021-11-12T10:00:00
+title: Highlights for the week ending November 12, 2021
+changes_categories:
+  - Highlights
+owner:
+  - https://github.com/orgs/giantswarm/teams/sig-product
+---
+
+## Apps
+
+[Flux](https://github.com/giantswarm/flux-app) v0.7.0 provides the flux toolkit version 0.21.0 and adds service discovery labels for Prometheus, so that metrics are scraped.
```

Comment:
```suggestion
[Flux](https://github.com/giantswarm/flux-app) v0.7.0 provides the Flux Toolkit version 0.21.0 and adds service discovery labels for Prometheus, so that metrics are scraped.
```

this is a proper noun as used in upstream website/docs